include/unifex/detail/type_erasure_builtins.hpp

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <unifex/this.hpp>
+
+#include <type_traits>
+
+#include <unifex/detail/prologue.hpp>
+
+//
+// This header contains some helper-CPOs for type-erasing wrappers that let us
+// create vtable entries for built-in operations like destructors and
+// copy/move constructors.
+//
+// These CPOs can be passed to the list of CPOs for a vtable<> to create vtable
+// entries for these operations. These CPOs are only customisable through the
+// builtin operations and not via tag_invoke().
```

Comment:
You customise the `destroy_cpo` by defining a destructor.
You customise the `move_construct_cpo` by defining a move constructor.
You customise the `copy_construct_cpo` by defining a copy-constructor.

Reply:
Ok, then I suggest rewording this part
> These CPOs are only customisable through the builtin operations and not via tag_invoke().

to say "special member functions" instead of "builtin operations", and/or just copy your comment above into the source.